lib/execution_engine2/utils/Condor.py

```diff
@@ -72,44 +80,16 @@ def get_default_client_group_and_requirements(self, client_group):
             section = client_group
             default_resources[self.CG] = client_group
         else:
-            section = self.config["DEFAULT"][self.DEFAULT_CLIENT_GROUP]
-            default_resources[self.CG] = section
+            client_group = self.config["DEFAULT"][self.DEFAULT_CLIENT_GROUP]
+            default_resources[self.CG] = client_group
+            section = client_group
```

Comment:
Which assignment?

Reply:
unless I read it wrong, `section = client_group` could be removed. Then in `default_resources[item] = self.config.get(section=section, option=item)`, we can just do `default_resources[item] = self.config.get(section=client_group, option=item)`

Reply:
True.
 I figured it would be more readable by explicitly saying I was using it as a section. I could go either way on this, let me know if you want me to change this.

Reply:
nahhhh. sure. we can leave it like this.